src/UserRepository.js

```diff
@@ -0,0 +1,18 @@
+const {userData} = require("../data/users")
+class UserRepository {
```

Comment:
We all agree that we will remove this practice once we have the knowledge of how to do so, the importation can be seen as a temporary approach.